src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+    multiply(number) {
+      this.result = this.result * number;
+
+      return this;
+    },
+    divide(number) {
+      this.result = this.result / number;
```

Comment:
There is a potential issue in the `divide` method. Dividing by zero will result in `Infinity` or `NaN`, which may not be the desired behavior. Consider adding a check to handle division by zero appropriately.